RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 - [internal] Refactored some API calls for fetching a Note, Product, and Product Review. 
 - Products: we improved our VoiceOver support in Product Price settings
 - In Settings > Experimental Features, a Products switch is now available for turning Products M2 features on and off for simple products (default off for beta testing). Products M2 features: update product images, product settings, viewing and sharing a product.
+- The WIP banner on the Products tab is now collapsed by default.
```

Comment:
Do you think that it can be useful for our users? I'm not sure they know what is the WIP banner but feel free to leave it, I believe we can decide what to do with it when reviewing the release notes.

Reply:
good question, I added it just because it's a user-facing change and possibly an improvement since there's more vertical space for the products tab content by default now (I updated the message to explain a bit why we made this change https://github.com/woocommerce/woocommerce-ios/pull/2300/commits/45c16051744008fb1dac1e6b0d07e4cd60809dc1). I think editorial team will decide whether this in the release notes, and we can review them later as well!